### PR TITLE
fix: pagination search button

### PR DESCRIPTION
**Pull request by alexbarnsley**
## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [ ] Ready to be merged

### CODE REVIEW

File: resources/views/pagination.blade.php [hunk 1]
```diff
@@ -80,11 +80,14 @@ class="py-2 px-3 w-full bg-transparent dark:text-theme-secondary-200"
             </form>
 
             <div class="hidden px-2 rounded bg-theme-primary-100 dark:bg-theme-secondary-800 md:flex flex-inline">
+                @php ($pageIndex = 1)
```
Comment (ItsANameToo) [inline resources/views/pagination.blade.php:83]: there's a `$loop->index` available in `foreach` loops

**Review by ItsANameToo — CHANGES_REQUESTED** [review]

File: resources/views/pagination.blade.php (hunk 1 above)
Comment (alexbarnsley) [reply resources/views/pagination.blade.php:83]: yeh but this is the result of 2 loops 🤔

Comment (alexbarnsley) [reply resources/views/pagination.blade.php:83]: e.g. `@php ($pageIndex++)` is inside a second loop

Comment (ItsANameToo) [reply resources/views/pagination.blade.php:83]: I see, alrighty then

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**